deepchecks/checks/integrity/string_mismatch_comparison.py

```diff
@@ -82,24 +82,24 @@ def __init__(
         self.ignore_columns = ignore_columns
         self.n_top_columns = n_top_columns
 
-    def run(self, dataset, baseline_dataset, model=None) -> CheckResult:
+    def run(self, train_dataset, test_dataset, model=None) -> CheckResult:
         """Run check.
 
         Args:
-            dataset (Dataset): A dataset object.
-            baseline_dataset (Dataset): A dataset object.
+            train_dataset (Dataset): A dataset object.
```

Comment:
I know it's not your code here, but please help us and copy the docstring from other place where the docs relate to the face that it's train and test

Reply:
updated